pkg/booleanpolicy/querybuilders/special_cases.go

```diff
@@ -148,6 +148,23 @@ func ForWriteableHostMount() QueryBuilder {
 	})
 }
 
+// ForFixable returns a specific query builder for whether a CVE is fixable or not.
+func ForFixable() QueryBuilder {
+	return queryBuilderFunc(func(group *storage.PolicyGroup) []*query.FieldQuery {
```

Comment:
should this also use `wrapForVulnMgmt`? That accounts for deferred vulns (deferred and FP vulns won't get selected)

Reply:
Yes, added